test/simulator/moves/ragepowder.js

```diff
@@ -27,15 +28,15 @@ describe('Rage Powder', function () {
 		let hitCount = [0, 0, 0];
 		battle.p1.active[0].damage = function (...args) {
 			hitCount[0]++;
-			return BattleEngine.BattlePokemon.prototype.damage.apply(this, args);
+			return BattlePokemon.prototype.damage.apply(this, args);
```

Comment:
IMO it would be preferably to use ``Object.getPrototypeOf(this)`` instead of reintroducing the battle engine require.